configs/katello_33.yaml

```diff
@@ -0,0 +1,178 @@
+---
+:project: katello
+:github_org: katello
+:prior_releases:
+  :3.2.3:
+    :redmine_release_id: 214
+:releases:
+  :3.3:
+    :redmine_release_id: 188
+:code_name: Baltic Porter
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-3.3
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :katello-selinux:
```

Comment:
don't think so, i just copied all this from 3.2's config file.  I don't think its needed (but i don't think some below it are either).